### PR TITLE
Start FAQ Page, Mostly with EV Questions

### PR DESCRIPTION
**Pull request by vkoves**
## Overview

Adds an FAQ page and links it in the header - this page answers a lot of the common questions I have gotten particularly about electric vehicles, and I plan on adding more info about renewable energy.

Starts on issue #30 but doesn't close it out yet. I'm opening this PR now since there's already a lot of text that might take us some time to discuss, but I think this starter FAQ is better than not having one.

### Demo

Check out the Netlify - this is mostly a giant text page

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- Load the Netlify and confirm you can go to `/faq` from the header
- Confirm the page is readable on desktop and mobile
- Actually read all the text.

### CODE REVIEW

File: src/pages/faq.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          answer is no. But make no mistake, heat pumps absolutely work in cold
+          climates. Not only that, but often they are the most energy efficient
+          and cost effective solution available. A few decades ago, most heat
+          pumps stopped working when the temperature dropped below 20 or 30
+          degrees fahrenheit. By contrast, today’s heat pumps can run more
+          efficiently than any other HVAC system all the way down to about -25
+          fahrenheit. Just ask the millions of homeowners in Scandinavia. People
+          in Norway, Finland and Sweden are installing heat pumps at a faster
+          pace than anywhere else in Europe.
+          <a href="https://carbonswitch.com/do-heat-pumps-work-in-cold-weather/">
```
Comment (derekeder) [inline src/pages/faq.js:35]: additional space needed before this link

File: src/pages/faq.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+          Do you have questions? We've compiled some of the most common ones we
+          get into this one page.
+        </p>
+
+        <h2>Heat Pumps</h2>
+
+        <h3>Do heat pumps work in colder climates?</h3>
+
+        <p>
+          Google this question or ask a contractor and you might think the
```
Comment (derekeder) [inline src/pages/faq.js:25]: I would axe this first sentence and begin with "Heat pumps absolutely ..."

File: src/pages/faq.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          and cost effective solution available. A few decades ago, most heat
+          pumps stopped working when the temperature dropped below 20 or 30
+          degrees fahrenheit. By contrast, today’s heat pumps can run more
+          efficiently than any other HVAC system all the way down to about -25
+          fahrenheit. Just ask the millions of homeowners in Scandinavia. People
+          in Norway, Finland and Sweden are installing heat pumps at a faster
+          pace than anywhere else in Europe.
+          <a href="https://carbonswitch.com/do-heat-pumps-work-in-cold-weather/">
+            Read more at CarbonSwitch
+          </a>
```
Comment (derekeder) [inline src/pages/faq.js:37]: I think we should also add:

"In colder climates, a potential hurdle to installing heat pumps is home insulation. You may want to get a [blower door test](https://www.energy.gov/energysaver/blower-door-tests) to determine how much heat you are leaking from your home"

File: src/pages/faq.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            Tom's Guide - EV Charging Explained
+          </a>
+        </p>
+
+        <h3>Why are EVs so expensive?</h3>
+
+        <p>
+          EVs require large battery packs, particularly if you are looking for a
+          very long range. However, it's worth keeping in mind that an electric
+          vehicles has lower fuel costs (electricity per mile is generally
```
Comment (derekeder) [inline src/pages/faq.js:128]: "vehicle"

File: src/pages/faq.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+          cheaper than gasoline) and lower maintenance costs, since electric
+          vehicles are much more simpler mechanically speaking and have lower
```
Comment (derekeder) [inline src/pages/faq.js:130]: "much simpler"

**Review by derekeder — CHANGES_REQUESTED** [review]
a few small changes to make and we're good to go!

**Comment by vkoves** [comment]
@derekeder - I responded to your feedback but I'm not at home to run the `yarn format` command. Do you mind running that on this branch and pushing it up?

**Review by derekeder — APPROVED** [review]
linter ran & tests pass. merging in!

**Pull request merged**